client/tests/integration/status_response.rs

```diff
@@ -0,0 +1,54 @@
+use std::str::FromStr as _;
+
+use eyre::Result;
+use iroha_client::data_model::prelude::*;
+use iroha_telemetry::metrics::Status;
+use parity_scale_codec::DecodeAll;
+use test_network::*;
+
+fn status_eq_excluding_uptime_and_queue(lhs: &Status, rhs: &Status) -> bool {
+    lhs.peers == rhs.peers
+        && lhs.blocks == rhs.blocks
+        && lhs.txs_accepted == rhs.txs_accepted
+        && lhs.txs_rejected == rhs.txs_rejected
+        && lhs.view_changes == rhs.view_changes
+}
+
+#[test]
+fn json_and_scale_statuses_equality() -> Result<()> {
+    let (_rt, network, client) = Network::start_test_with_runtime(1, Some(10_945));
```

Comment:
Port is overlapping with `multisignature_transactions_should_wait_for_all_signatures`, please pick free port.

Reply:
@VAmuzing usually i just search for max port and than add 5 ports for every peer in this test, e.g. if previous port was `10_000` with 4 peers, next port for the test would be `10_000 + (4 + 1) * 5 = 10_025`